service/src/main/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneResourcesFlight.java

```diff
@@ -66,13 +67,14 @@ public CreateLandingZoneResourcesFlight(FlightMap inputParameters, Object applic
     parametersResolver =
         new ParametersResolver(landingZoneRequest.parameters(), LandingZoneDefaultParameters.get());
 
-    addCreateSteps();
+    addCreateSteps(flightBeanBag.getAzureConfiguration());
   }
 
-  private void addCreateSteps() {
+  private void addCreateSteps(LandingZoneAzureConfiguration azureConfiguration) {
     stepsDefinitionProvider
         .get(
             armManagers,
+            initializeAdminSubscriptionArmManager(azureConfiguration),
```

Comment:
I think it would be good to keep all the managers under one umbrella if we need this separate instance with different configuration. What if we split them into two or more groups in ArmManagers?

Reply:
A fair point, I'll give that a try.